src/emulator/config/include/config/config.h

```diff
@@ -43,15 +42,34 @@ struct Config {
     bool pstv_mode = false;
     bool show_gui = false;
     bool show_game_background = true;
-    std::string pref_path = {};
-    fs::path config_path = {};
-    bool overwrite_config = true;
-    bool load_config = false;
+    int icon_size = 64;
     bool archive_log = false;
     bool texture_cache = true;
-    optional<bool> wait_for_debugger;
+    int sys_button = 1;
+    int sys_lang = 1;
     std::string background_image = {};
     float background_alpha = 0.300f;
-    int icon_size = 64;
+    int log_level = 0;
+    std::string pref_path = {};
+    fs::path config_path = {};
+    bool overwrite_config = true;
+    bool load_config = false;
     bool discord_rich_presence = true;
-};
+    optional<bool> wait_for_debugger;
+
+    // void merge_config(const Config &cfg, const std::string &new_pref_path = {}) const {
+    //     this = std::move(cfg);
+    //     this->pref_path = new_pref_path;
+    // }
+
+    // Optimal based on: http://stackoverflow.com/a/24477023
+    void merge_lle(std::vector<std::string> new_lle) const {
```

Comment:
Maybe make an function in util? Maybe useful in the future

Reply:
Done, and made it templated. It was a pretty quick and good idea, anyway.

For convenience, it is over in `util/vector_utils.h`.